README.md

```diff
@@ -1,22 +1,27 @@
 # ambit
 
+[![Rust](https://github.com/plamorg/ambit/actions/workflows/rust.yml/badge.svg)](https://github.com/plamorg/ambit/actions/workflows/rust.yml)
+
 Dotfile manager written in Rust.
 
-(Planned) features:
+`ambit` symbolically links files from a git repository to your system.
+Files can be easily synced and are managed through a custom configuration file.
+
+Features:
 
+*   Custom configuration syntax
```

Comment:
Try promoting it a bit more: `Custom configuration syntax, specifically tailored towards dotfile managing` :)